New in chrome 106 post (#3840)
GoogleChrome/developer.chrome.com: site/en/blog/new-in-chrome-106/index.md

---
title: New in Chrome 106
description: >

>   Chrome 106 is rolling out now. There are new Intl APIs to give you more control when formatting numbers. There’s an origin trial for the new Pop Up API, making it easy to surface critical content to the user. There are a handful of CSS improvements. And there’s plenty more

Reviewer: ```suggestion
  Chrome 106 is rolling out now. There are new Intl APIs to give you more control when formatting numbers. There’s an origin trial for the new Pop Up API, making it easy to surface critical content to the user. There are a handful of CSS improvements. And there’s plenty more.
```

but also the flexibility to modify the default behaviors. Using only CSS and HTML.

Check out Jhey’s [article](/blog/pop-ups-theyre-making-a-resurgence/) for more examples and API options.

[Sign up for the origin trial](/en/docs/web-platform/origin-trials/) to easily give your users timely information. Let us know what you think.

## New css features {: #css-interop }

There are two new CSS features that improve interop and hopefully make your life a little easier.

There is a new length unit in town: `ic` is joining the party. `ic` is similar to `ch`.
But `ic` is used specifically for text written in languages that use ideograms,
basically it measures length based on the width or height of this character [point somewhere] which means water.

It is a unit designed to size text, allowing you to limit width to improve readability, and it gives predictable control regardless of the text size.

For example if you set the `max-width` of a container, let's say to 10ic,
you *know* that container will contain at most 10 full width glyphs,
no matter the font-size. Check it out in the following example:

{% Codepen {

>     user: 'tropicadri',
>     id: 'OJZmgOg',

Reviewer: ```suggestion
    user: 'web-dot-dev',
    id: 'eYrybaj',
```

    height: 450,
    tab: 'result'
  }
%}

> CSS Grid support for interpolation for `grid-template-columns` and `grid-template-rows` is coming, it was planned for 106, but is delayed and will be launched in Chrome 107, you can still try it out in Chrome Beta. Here is Bramu’s code as an example:

Reviewer: ```suggestion
CSS Grid support for interpolation for `grid-template-columns` and `grid-template-rows` is coming, it was planned for 106, but is delayed and will be launched in Chrome 107, you can still try it out in Chrome Beta. Here is Bramus' code as an example:
```

{% Codepen {

>     user: 'bramus',
>     id: 'JjvEExW',

Reviewer: ```suggestion
    user: 'web-dot-dev',
    id: 'XWqVowx'
```

>     id: 'XWqVowx'

Author: ```suggestion
    id: 'XWqVowx',
```
